ser574-green-team/taigit
# T165 find pom

To test,
1. add another argument to all axios calls : {"auth": {"username": "yourgithubusername", "password": "yourauth"}}
2. put stuff like `{console.log(GH.getBuilds("ser574-green-team", "taigit"))}` in some part of the body of GitHub.js in taigit/taigit/src/components. Run the program, go to the github page, and inspect element.

There are 6 functions, usesMaven, usesGradle, usesCMake, usesAnt, usesMake, and getBuilds.
**Due to the high volume of requests that these functions make, I recommend only using getBuilds.**
Call getBuilds with repos, and check to see that you get the expected result: it should return an array of strings containing all build systems used in that repo (hopefully just one). 
It should return an empty list for taigit, as that uses none of these build systems.
